core/domain/stats_domain.py

```diff
@@ -1450,7 +1395,7 @@ def __init__(
                 LearnerAnswerInfo objects.
             accumulated_answer_info_json_size_bytes: int. The size of
                 learner_answer_info_list in bytes.
-            schema_version: int. The schema version of the
+            learner_answer_info_schema_version: int. The schema version of the
                 LearnerAnswerInfo class.
```

Comment:
... LearnerAnswerInfo dicts stored in this instance.

Reply:
Done